### PR TITLE
feat: error pages

### PR DESCRIPTION
**Pull request by faria-s**

### CODE REVIEW

**Review by AfonsoMartins26 — APPROVED** [review]
Good work!👍

File: config/dev.exs [hunk 1]
```diff
@@ -22,7 +22,7 @@ config :cesium_link, CesiumLinkWeb.Endpoint,
   http: [ip: {127, 0, 0, 1}, port: 4000],
   check_origin: false,
   code_reloader: true,
-  debug_errors: true,
+  debug_errors: false,
```
Comment (ruilopesm) [inline config/dev.exs]: Undo this change before merging 🙏

Comment (faria-s) [reply config/dev.exs]: Done

File: lib/cesium_link_web/controllers/error_html/404.html.heex [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    </script>
+  </head>
+  <body>
+    <main class="my-60 flex-auto">
+      <section class="flex w-full flex-col items-center justify-center">
+        <a href="/" class="items-center justify-center">
+          <img class="my-5 w-40" src="/images/cesium.svg" alt="Cesium-Logo" />
+        </a>
+        <p class="text-center text-5xl font-medium">
+          404 Page not found
```
Comment (ruilopesm) [inline lib/cesium_link_web/controllers/error_html/404.html.heex]:
```suggestion
          404 Page Not Found
```

File: lib/cesium_link_web/controllers/error_html/404.html.heex [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <title>404</title>
+    <link rel="stylesheet" href="/assets/app.css" />
+    <script defer type="text/javascript" src="/assets/app.js">
+    </script>
+  </head>
+  <body>
+    <main class="my-60 flex-auto">
+      <section class="flex w-full flex-col items-center justify-center">
+        <a href="/" class="items-center justify-center">
+          <img class="my-5 w-40" src="/images/cesium.svg" alt="Cesium-Logo" />
```
Comment (ruilopesm) [inline lib/cesium_link_web/controllers/error_html/404.html.heex]:
```suggestion
          <img class="my-5 w-40" src="/images/cesium.svg" alt="cesium-logo" />
```

File: lib/cesium_link_web/controllers/error_html/404.html.heex [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        </a>
+        <p class="text-center text-5xl font-medium">
+          404 Page not found
+        </p>
+        <p class="my-3 text-center text-2xl">Oops! Looks like you are lost.</p>
+        <p class="my-5 text-center text-lg">Click on the logo to turn back.</p>
```
Comment (ruilopesm) [inline lib/cesium_link_web/controllers/error_html/404.html.heex]:
```suggestion
        <p class="my-5 text-center text-lg">Click on the logo to return home.</p>
```

File: test/cesium_link_web/controllers/error_html_test.exs [hunk 5]
```diff
@@ -5,7 +5,7 @@ defmodule CesiumLinkWeb.ErrorHTMLTest do
   import Phoenix.Template
 
   test "renders 404.html" do
-    assert render_to_string(CesiumLinkWeb.ErrorHTML, "404", "html", []) == "Not Found"
+    assert render_to_string(CesiumLinkWeb.ErrorHTML, "404", "html", []) =~ "404 Page not found"
```
Comment (ruilopesm) [inline test/cesium_link_web/controllers/error_html_test.exs]:
```suggestion
    assert render_to_string(CesiumLinkWeb.ErrorHTML, "404", "html", []) =~ "404 Page Not Found"
```

Comment (faria-s) [reply test/cesium_link_web/controllers/error_html_test.exs]: Done

File: lib/cesium_link_web/controllers/error_html/404.html.heex (hunk 4 above)
Comment (faria-s) [reply lib/cesium_link_web/controllers/error_html/404.html.heex]: Done

File: lib/cesium_link_web/controllers/error_html/404.html.heex (hunk 3 above)
Comment (faria-s) [reply lib/cesium_link_web/controllers/error_html/404.html.heex]: Done

**Comment by faria-s** [comment]
Waiting for the tests to run, missing a commit. On it

**Comment by faria-s** [comment]
Ok, Done

**Review by ruilopesm — APPROVED** [review]
Excellent work 🙌🏼

**Review by joaodiaslobo — APPROVED** [review]

**Review by AfonsoMartins26 — APPROVED** [review]
Great job! 🚀
